imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -40,6 +45,19 @@ Template.CoreNavigationBar.helpers({
       // }
     };
   },
+  notificationButtonComponent() {
+    return {
+      component: NotificationContainer
+      // icon: "fa fa-bell",
```

Comment:
Yes, it should be an empty container. It sends data to its representational component. `NotificationComponent`